src/styles/block/calendar.scss

```diff
@@ -0,0 +1,42 @@
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gapsize;
+  max-width: $width-cal;
+  padding: 20px;
+
+  @each $day, $number in $daynumber {
+    &--start-day--#{$day} :nth-child(1) {
+      margin-left: ($size-char + $gapsize) * $number;
+    }
+  }
+
+  @for $indexDay from 28 through 31 {
+    &--month-length--#{$indexDay} :nth-child(n + #{$indexDay + 1}) {
+      display: none;
+    }
+  }
+
+  &__day {
+    box-sizing: border-box;
```

Comment:
it is a good practice to add `box-sizing: border-box;` on all elements using `*`